lib/dungeonExpWeight.js

```diff
@@ -14,18 +14,19 @@ const getDungeonExpWeight = cataXP => {
     }
 
     let n;
+    let extra;
     let tempLevel = 0;
     if (cataXP < 569809640)
         n = 0.2 * (level / 50) ** 1.538679118869934;
     else {
         let part = 142452410;
         tempLevel = 50 + (cataXP - dungeonMaxXP) / part;
-        n = (1 + ((tempLevel - 50) / 50)) ** 2.967355422;
+        extra = (500 * ((exp - lv50) / part) ** (1/1.781925776625157));
```

Comment:
```suggestion
        extra = (500 * ((cataXP - dungeonMaxXP) / part) ** (1/1.781925776625157));
```

Reply:
`exp` and `lv50` are not defined in this scope

Reply:
oops thank you